content/en/news/security/istio-security-2021-003/index.md

```diff
@@ -0,0 +1,127 @@
+when using [mTLS-only fields](/docs/concepts/security/#dependency-on-mutual-tls) in the authorization policy.
+
+You can stop reading if
+
+- Your authorization policy does not use [mTLS-only fields](/docs/concepts/security/#dependency-on-mutual-tls); or
+
+- Your authorization policy uses mTLS-only fields and you have also enabled mTLS with STRICT mode or your authorization
+policy is configured to reject plain text traffic explicitly.
+
+## Issue
+
+In authorization policy, the following are [mTLS-only fields](/docs/concepts/security/#dependency-on-mutual-tls):
+
+- the `principals` and `notPrincipals` field under the `source` section
+- the `namespaces` and `notNamespaces` field under the `source` section
+- the `source.principal` custom condition
+- the `source.namespace` custom condition
+
+These mTLS-only fields will never match when the traffic is plain text (non mTLS) and the request might be allowed unexpectedly.
+
+The following is an example ALLOW policy that uses mTLS-only fields to allow requests if it is not from the namespace `foo`:
+
+{{< text yaml >}}
+apiVersion: "security.istio.io/v1beta1"
+kind: "AuthorizationPolicy"
+metadata:
+  name: allow-ns-not-foo
+spec:
+  action: ALLOW
+  rules:
+  - from:
+    - source:
+        notNamespaces: ["foo"]
+{{< /text >}}
+
+A **plain text request** from the namespace `foo` will actually be allowed. The mTLS-only field `notNamespaces` will be
+compared to an empty value when mTLS is not used. The comparison result is always matched that makes the policy to allow
+the request as if it was not from the namespace `foo`.
+
+The following is an example DENY policy that uses mTLS-only fields to reject a request if it is from the namespace `foo`:
+
+{{< text yaml >}}
+apiVersion: "security.istio.io/v1beta1"
+kind: "AuthorizationPolicy"
+metadata:
+  name: reject-ns-foo
+spec:
+  action: DENY
+  rules:
+  - from:
+    - source:
+        namespaces: ["foo"]
+{{< /text >}}
+
+A **plain text request** from the namespace `foo` will not be rejected. The mTLS-only field `namespaces` will be
+compared to an empty value when mTLS is not used. The comparison result is always not matched that makes the policy to
+not reject the request as if it was not from the namespace `foo`.
+
+## Solution
+
+To solve this problem, it's recommended to always [enable mTLS with STRICT mode](/docs/tasks/security/authentication/authn-policy/#enable-mutual-tls-per-namespace-or-workload)
+on the workloads before using any mTLS-only fields in the authorization policy on the same workload.
+
+If you are unable to enable mTLS with STRICT mode for the workload, the alternative solution is to update the authorization
+policy to explicitly allow traffic with non-empty namespaces or reject traffic with empty namespaces (`*` implies non-empty and `not *` implies empty).
+As namespace can only be extracted when mTLS is STRICT. The policies below effectively also reject any plain text traffic.
```

Comment:
I think you can either remove the `As namespace...` sentence (it's not a full sentence), or end the prior sentence with a comma and lowercase As. That does make the prior sentence a little too long.  Maybe:

```
If you are unable to enable mTLS with STRICT mode for the workload, the alternative solution is
to update the authorization policy to explicitly allow traffic with non-empty namespaces or
reject traffic with empty namespaces, as namespace can only be extracted when mTLS is 
STRICT. `*` implies non-empty namespaces and `not *` implies empty namespaces. The 
policies below also reject any plain text traffic.
```